src/components/partials/profile/SendTokenModal.vue

```diff
@@ -1,7 +1,28 @@
 <template>
     <Modal :open="open" @update:open="closeSendToken()">
         <template #title> Transfer Token </template>
-        <template #body>
+        <template v-if="isTransferSuccessful || isTransferFail" #body>
+            <div v-if="isTransferSuccessful" class="mb-4">
+                <Alert
+                    :isSuccess="true"
```

Comment:
you are not using isSuccess in Alert.vue -> why not just use isError true or false? Because otherwise what would be if isSuccess and isError are set to true? I think just having one of them makes more sense.